cmd/pilot-discovery/main.go

```diff
@@ -199,6 +210,9 @@ func init() {
 	discoveryCmd.PersistentFlags().StringVar(&flags.consulargs.serverURL, "consulserverURL", "",
 		"URL for the consul server")
 
+	_ = discoveryCmd.PersistentFlags().MarkHidden("namespace")
+	_ = discoveryCmd.PersistentFlags().MarkHidden("profile")
+	_ = discoveryCmd.PersistentFlags().MarkHidden("discovery_cache")
```

Comment:
why `_ = ...` instead of just calling the function and ignoring the return? Or capturing the return

Reply:
linter complains.